spec/support/cuprite/browser_logger.rb

```diff
@@ -2,7 +2,7 @@
 
 using Rainbow
 
-class CupriteLogger
+class Cuprite::BrowserLogger
```

Comment:
Since I already had this Cuprite-related class, and since I was going to create another Cuprite-related file (to register the domain-restricted driver), I wanted to create a `spec/support/cuprite/` directory, within which both could be organized. So that's why this constant name has changed. I also felt that `BrowserLogger` is likely than `Logger` to possibly conflict with a constant that might already be within the gem's `Cuprite` namespace.